pkgs/development/tools/build-managers/bazel/default.nix

```diff
@@ -396,7 +397,7 @@ stdenv.mkDerivation rec {
       # So we hardcode it to something useful.
       substituteInPlace \
         src/main/java/com/google/devtools/build/lib/bazel/rules/BazelRuleClassProvider.java \
-        --replace /bin:/usr/bin ${defaultShellPath}
+        --replace /bin:/usr/bin:/usr/local/bin ${defaultShellPath}
```

Comment:
This patches the following line:
https://github.com/bazelbuild/bazel/blob/master/src/main/java/com/google/devtools/build/lib/bazel/rules/BazelRuleClassProvider.java#L472

I assume there once was only `/bin:/usr/bin` in there.

To avoid silent regressions in the future, we should use a patch replacing this with a placeholder, and substitute the placeholder before applying the patch, as done in `https://github.com/NixOS/nixpkgs/blob/master/pkgs/os-specific/linux/udisks/2-default.nix#L22`.

Reply:
Done.